src/devices/machine/mc68681.cpp

```diff
@@ -474,6 +530,17 @@ READ8_MEMBER( sc28c94_device::read )
 	return r;
 }
 
+READ8_MEMBER( xr68c681_device::read )
+{
+	if (offset == 0x02)
+	{
+		LOG( "Reading XR68C681 (%s) reg 0x02 (MISR)\n", tag());
```

Comment:
Spacing is out on this line.  Also please don't put `tag()` in log messages - `logerror` (the default output sink for `LOG`) already prefixes lines with the device tag.